extension/content/script.js

```diff
@@ -150,6 +161,17 @@ async function initUI() {
   $("#compare-configs").addEventListener("click", changeTabs);
 
   $("#changed-sections").addEventListener("change", displayDiff);
+  $("body > form").addEventListener("keypress", (e) => {
+    if (e.keyCode == "13") {
```

Comment:
Enter on a form will be triggering submit right? would it be a little cleaner to be listening for submit as opposed to keypress?

Reply:
I did try submit, but for some reason that didn't work. I suspect it is because we don't have a submit handler / button on the form itself.